addons/mail/models/mail_link_preview.py

```diff
@@ -16,6 +16,7 @@ class LinkPreview(models.Model):
     _description = "Store link preview data"
 
     message_id = fields.Many2one('mail.message', string='Message', index=True, ondelete='cascade', required=True)
+    is_hidden = fields.Boolean()
```

Comment:
```suggestion
    is_hidden = fields.Boolean(default=False)
```

Reply:
Add that with a default empty so we don't need to make an upgrade script to add the value. And to reduce the data we need to store in the database.

Reply:
This is a new field and the PR target master, why would you need an upgrade script? The new field should just initialize everything to `false`.
I m missing something?

Reply:
We don't need to store the value false in the database, we just need the information when `is_hidden=true` and in this case, undefined is the same as false, and we don't need to store those values in the database.